pkg/archive/changes.go

```diff
@@ -18,14 +18,19 @@ import (
 	"github.com/docker/docker/pkg/system"
 )
 
+// ChangeType represents the change type.
 type ChangeType int
 
 const (
+	// ChangeModify represents the modify operation.
 	ChangeModify = iota
+	// ChangeAdd represents the add operation.
 	ChangeAdd
+	// ChangeDelete represents the delete operation.
 	ChangeDelete
 )
 
+// Change represents a change, it wraps the change type and path.
```

Comment:
Add maybe a little more info of what is a change and when it is used (mainly if not only for the layer diff stuff).